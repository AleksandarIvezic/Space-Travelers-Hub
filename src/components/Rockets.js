import React from 'react';

const Rockets = () => (
  <div>
    <h3>Rockets</h3>
  </div>
);

export default Rockets;
