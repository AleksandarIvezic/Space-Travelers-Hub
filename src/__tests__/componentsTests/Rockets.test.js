import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../redux/storeConfig';
import Rockets from '../../components/Rockets';

describe('Test if Rockets', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rockets />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
