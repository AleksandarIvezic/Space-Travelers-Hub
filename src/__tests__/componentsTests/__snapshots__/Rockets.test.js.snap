// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if Rockets renders correctly 1`] = `
<div>
  <ul />
</div>
`;
